libs/beacon/src/beacon_setup_service.cpp

```diff
@@ -86,48 +91,47 @@ uint64_t GetTime()
   return static_cast<uint64_t>(std::time(nullptr));
 }
 
+//#define CHANNEL_TYPE RBC
+#define CHANNEL_TYPE muddle::RBC
```

Comment:
This should be refactoring into a factory

Reply:
OK